src/templates/event.pug

```diff
@@ -37,7 +37,7 @@ block content
 								!= contents
 						if moment(endDate).add(14,'days').isAfter(moment(),'day') && contactEmail
 							.event-basic-info-contact
-								| For more information email
+								| For more information email&#32;
```

Comment:
This is fine, also you can use a string:

```pug
.thing
  | test text here#{' '}
```

Not necessarily nicer-looking, but makes it clearer what the intent is